docs/matchers/webhook.md

```diff
@@ -56,3 +56,19 @@ class MySkill(Skill):
         # Send a custom aiohttp.web.Response object back to the webhook
         return Response(body='my custom response', status=201)
 ```
+## Securing Webhooks
+
+You can also secure the webhooks by adding an optional `webhook-token` value to the `web` configuration. This enables a token-based authentication where the `POST` request is required to have an `Authorization` header with the bearer token as its value.
+
+**Example Config**
+
+```yaml
+web:
+  webhook-token: "aabbccddee"
+```
+
+**Example POST Request Header**
+
+```
+Authorization: Bearer aabbccddee
```

Comment:
Could you add an example `curl` command and/or Python `requests` command showing how to use it?

Reply:
Added `curl` command.